etc/scripts/changelog.py

```diff
@@ -0,0 +1,61 @@
+import urllib
+import urllib2
+from datetime import date
+from dateutil.relativedelta import relativedelta
+import traceback
+import json
+
+
+def format_commit(commit):
```

Comment:
Rather than doing this here. 

Why not just dump the commits into json. Then use pug?